components/tools/OmeroWeb/build.xml

```diff
@@ -59,6 +59,21 @@
                 </touch>
             </then>
         </if>
+        <if><not><available file="${basedir}/../target/lib/python/clogging"/></not>

```

Comment:
Do you prefer if all web dependencies, like django pipeline, etc. will go to fallback?